### ISSUE TITLE
Unused variables in hgt_nc.py

### ISSUE THREAD

**Issue by CongWeilin** [awslabs/graphstorm#177]
Hi, I found the following when playing with the example codes:

- In file [`examples/customized_models/HGT/hgt_nc.py`](https://github.com/awslabs/graphstorm/blob/main/examples/customized_models/HGT/hgt_nc.py), the final linear projection [`self.out`](https://github.com/awslabs/graphstorm/blob/696ca2571565322360d2d5c80d9d591eba5d4cd9/examples/customized_models/HGT/hgt_nc.py#L184) is not used. Might need to add `self.out(h[self.target_ntype])` to [`h[self.target_ntype]`](https://github.com/awslabs/graphstorm/blob/696ca2571565322360d2d5c80d9d591eba5d4cd9/examples/customized_models/HGT/hgt_nc.py#LL205C35-L205C55) before the loss function

- The `self.configs.lr` does not affect the learning rate of an optimizer. Instead, it was controlled by the default variable `lr` in [`create_optimizer(self, lr=0.001)`](https://github.com/awslabs/graphstorm/blob/696ca2571565322360d2d5c80d9d591eba5d4cd9/examples/customized_models/HGT/hgt_nc.py#LL240C9-L240C25)

**Comment by zhjwy9343** [comment]
Has been fixed in the PR #179.
